### ISSUE TITLE
Add missing pagebreak to COBOL Syntax

### ISSUE THREAD

**Issue by spgennard** [spgennard/vscode_cobol#196]
```cobol
      / wibble
      *  stuff
      *> wibble
      *
012@22* stuff
121120* wibble
```

**Comment by spgennard** [comment]
new scope for ""comment.line.cobol.newpage" which is by default a comment but you can tweak it to make the newpage break comment to standout.

```json
    "editor.tokenColorCustomizations": {
        "textMateRules": [
            {
                "scope": "comment.line.cobol.newpage",
                "settings": {
                    "foreground": "#ffff00",
                    "fontStyle": "italic bold underline",
                }
            }
        ]
   ```

For example:

![image](https://user-images.githubusercontent.com/5991192/92528527-d7b41500-f220-11ea-8e8d-200b6545b55b.png)

**Comment by GitMensch** [comment]
I really like that, what do you think about making it underlined by default?

**Comment by GitMensch** [comment]
Stranglely I have the leading spaces colored the same way - that happens when there's no "content" in, I _guess_ that isn't by design? If it is: can I request to only color the `/`in this case?

```cobol
           EXIT.
      /
      *---------------------------------------------------------------*
```
![sample view](https://user-images.githubusercontent.com/6699539/92560530-e0353b80-f272-11ea-93e7-2b7742b7ae86.png)

**Comment by spgennard** [comment]
I will review this later.

**Comment by spgennard** [comment]
You will need to say what theme you are using because I get:

![image](https://user-images.githubusercontent.com/5991192/92633255-6b8deb80-f2ca-11ea-9173-eae2926b0183.png)

**Comment by GitMensch** [comment]
Hm, no theming options ever set. But that was vscode 1.48, maybe that's different with 1.49.

**Comment by GitMensch** [comment]
Fixed with 00ac2e8246e30af21f60ce3ffd04384941d76f59.
